test/fuzz/p2p/secret_connection/init-corpus/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+func main() {
+	baseDir := flag.String("base", ".", "the base directory in which the corpus directory resides")
+	flag.Parse()
+
+	log.SetFlags(0)
+	InitCorpus(*baseDir)
+}
+
+func InitCorpus(baseDir string) {
```

Comment:
nit: some of these are private whereas others are public. Let's be consistent if possible.